unstructured/ingest/cli/utils.py

```diff
@@ -12,39 +23,35 @@ def conform_click_options(options: dict):
             options[k] = list(v)
 
 
-class DelimitedString(click.ParamType):
-    name = "delimited-string"
-
-    def __init__(self, delimiter: str = ",", choices: t.Optional[t.List[str]] = None):
-        self.choices = choices if choices else []
-        self.delimiter = delimiter
-
-    def convert(
-        self,
-        value: t.Any,
-        param: t.Optional[click.Parameter],
-        ctx: t.Optional[click.Context],
-    ) -> t.Any:
-        # In case a list is provided as the default, will not break
-        if isinstance(value, list):
-            split = [str(v).strip() for v in value]
-        else:
-            split = [v.strip() for v in value.split(self.delimiter)]
-        if not self.choices:
-            return split
-        choices_str = ", ".join(map(repr, self.choices))
-        for s in split:
-            if s not in self.choices:
-                self.fail(
-                    ngettext(
-                        "{value!r} is not {choice}.",
-                        "{value!r} is not one of {choices}.",
-                        len(self.choices),
-                    ).format(value=s, choice=choices_str, choices=choices_str),
-                    param,
-                    ctx,
-                )
-        return split
+def extract_configs(
+    data: dict,
+    validate: t.Optional[t.List[t.Type[BaseConfig]]] = None,
```

Comment:
okay so validation is "can I instantiate from data dictionary" and the `validate` argument is actually a list of Config subclasses to validate? I'd probably name something clearer. maybe validation_config_classes? other ideas?
also probably worth a one-liner on what we're doing here.